tests/test_minimal/test_metadata_schema.py

```diff
@@ -0,0 +1,201 @@
+                name="Fluorescence",
+                PlaneSegmentationChan1Plane0=dict(
+                    raw=dict(name="RoiResponseSeriesChan1Plane0", description="raw fluorescence signal"),
+                    deconvolved=dict(name="DeconvolvedChan1Plane0", description="deconvolved fluorescence signal"),
+                    neuropil=dict(name="NeuropilChan1Plane0", description="neuropil fluorescence signal"),
+                ),
+                PlaneSegmentationChan1Plane1=dict(
+                    raw=dict(name="RoiResponseSeriesChan1Plane0", description="raw fluorescence signal"),
+                    deconvolved=dict(name="DeconvolvedChan1Plane0", description="deconvolved fluorescence signal"),
+                    neuropil=dict(name="NeuropilChan1Plane0", description="neuropil fluorescence signal"),
+                ),
+            ),
+            DfOverF=dict(
+                name="DfOverF",
+                PlaneSegmentationChan1Plane0=dict(
+                    dff=dict(name="RoiResponseSeriesChan1Plane0", description="Array of df/F traces."),
+                ),
+                PlaneSegmentationChan1Plane1=dict(
+                    dff=dict(name="RoiResponseSeriesChan1Plane0", description="Array of df/F traces."),
+                ),
+            ),
+            SegmentationImages=dict(
+                name="SegmentationImages",
+                PlaneSegmentationChan1Plane0=dict(
+                    raw=dict(name="PlaneSegmentationChan1Plane0", description="raw segmentation image"),
+                    neuropil=dict(name="PlaneSegmentationChan1Plane0", description="neuropil segmentation image"),
+                ),
+            ),
+        ),
+    )
+
+    validate_metadata(metadata=metadata, schema=metadata_schema)
+
+
+device_error = "'Devices' is a required property"
+invalid_plane_name = "plane_segmentation_chan1_plane0"
+name_error = "'name' is a required property"
+description_error = "'description' is a required property"
+plane_name_error = f"'{invalid_plane_name}' does not match any of the regexes"
+not_enough_properties_error = "does not have enough properties"
+
+
+def test_invalid_ophys_metadata():
+    metadata_schema = load_dict_from_file(
+        Path(__file__).parent.parent.parent / "src" / "neuroconv" / "schemas" / "metadata_schema.json"
+    )
+
+    def create_valid_plane_name(name: str, number=0):
+        return f'{"".join(map(lambda str: str.capitalize(), name.split("_")))[:-1]}{str(number)}'
+
+    metadata = dict(
+        NWBFile=dict(
+            session_start_time="2020-01-01T00:00:00",
+            session_description="Auto-generated by neuroconv",
+            identifier="1234",
+        ),
+        Ophys=dict(
+            Fluorescence={
+                "name": "Fluorescence",
+                invalid_plane_name: dict(),  # Unchecked
+                create_valid_plane_name(invalid_plane_name): dict(),
+                create_valid_plane_name(invalid_plane_name, 1): dict(
```

Comment:
```suggestion
                "plane_segmentation_chan1_plane0": dict(),  # Unchecked
                "PlaneSegmentationChan1Plane0": dict(),
                "PlaneSegmentationChan1Plane1": dict(
```

I highly recommend making this aspect of the tests explicit so that a future developer can see at a glance what values are being asserted without having to scroll to the find the variable `invalid_plane_name` defined above, or having to understand how the `create_valid_plane_name` function sanitizes it